email_automation/email-config.py

```diff
@@ -74,10 +74,8 @@ def get_verified_arn(sts_client):
 def email_extract():
     with open (config_path) as config_file:
         data = json.load(config_file)
-        intro = data['intro']
-        emails = [i.strip() for i in intro['admin_users'].split(",")]
         admindash_prefs = data['admin_dashboard']
-        #maybe set a bool and if this is true, then add a section to the HTML. Otherwise leave it. Need to do the same for map trip lines tomorrow.
+        emails = [i.strip() for i in admindash_prefs['admin_access'].split(",")]
```

Comment:
**future cleanup**: can you add a link to the discussion around the choice of this location for the admin users?

Reply:
> **future cleanup**: can you add a link to the discussion around the choice of this location for the admin users?
Sent on teams; here is how the form will look from [Abby's PR](https://github.com/e-mission/nrel-openpath-deploy-configs/pull/49#issuecomment-1843223668)